pkg/proto/dapr/v1/dapr.proto

```diff
@@ -78,19 +78,19 @@ message InvokeServiceResponse {
 }
 
 message DeleteStateEnvelope {
-  string storeName = 1;
+  string state_name = 1;

```

Comment:
This should be  store_name everywhere. Its the store name for the store when  multiple state stores are used.

Reply:
nice catch. Yeah that's my bad :) I fixed it.